kvmagent/kvmagent/plugins/mini_fencer.py

```
@@ -66,7 +66,7 @@ def inner(*args, **kwargs):
 def getstatusoutput(c):
     # type: (str) -> (int, str)
     start_time = time.time()
-    r, o = commands.getstatusoutput(c)
+    r, o = subprocess.getstatusoutput(c)
```

Ruff (0.14.7)

Starting a process with a shell, possible injection detected

(S605)